defaults/main.yml

```diff
@@ -29,8 +29,8 @@ github_api_url: "https://api.github.com"
 # Personal Access Token for your GitHub account
 access_token: "{{ lookup('env', 'PERSONAL_ACCESS_TOKEN') }}"
 
-# Is it the runner for organization or not?
-runner_org: no
+# Is it the runner for a "repository", an "organization" or an "enterprise"?
+runner_type: "repository"
```

Comment:
The problem with out right removal of `runner_org` is that you will require all existing users that have set it change their code.

I'm not sure what @adamc00 thinks but you may want something like:
```
# Is it the runner for a "repository", an "organization" or an "enterprise"?
runner_type: "{{ (runner_org | default('no')) ? "organization" : "repository }}"
```

Reply:
Thanks @martin-andrew-ellis 
I've applied the changes discussed, following the example you provided.
The bulk of which is in `tasks/asserts.yml`